frontend/src/app/shared-components/shared-components.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+import { ToolbarComponent } from './toolbar/toolbar.component';
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { MatIconModule } from '@angular/material/icon';
+import { SharedComponentsRoutingModule } from './shared-components-routing.module';
+import { CommonModule } from '@angular/common';
+
+@NgModule({
+  declarations: [ToolbarComponent],
+  exports: [ToolbarComponent],
+  imports: [MatToolbarModule, MatIconModule, SharedComponentsRoutingModule, CommonModule],
```

Comment:
**chore:** remove SharedComponentsRoutingModule as this is not a routable module

Reply:
I removed the SharedComponentsRoutingModule as it was unnecessary. However, I do need to import the RouterModule for the `[routerLink]` in the toolbar component to work. Is this necessary or is there a way around it?

Reply:
That's alright